language/tools/move-cli-with-df/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
BTW, the reason why I think it's necessary to introduce a separate crate is native functions. In theory, error code maps or Move source code can be specified via command line options, or a config file if we find that tedious, but not native functions.